mobify/platform-scaffold
# [Integration Manager] Cleanups in preparation for the checkout schema

This PR removes some unneeded pieces of the checkout code in preparation for schematizing it.

 **JIRA**: Preparation for https://mobify.atlassian.net/browse/WEBDATA-69

## Changes
- No longer parse the shipping form name, instead hard-coding it in the form definition.
- Remove unused parsers and actions
- Re-arrange some of the notification code for the checkout shipping page.

## How to test-drive this PR
- Preview on Merlin's
- Run through the checkout
- See that nothing has changed
